modules/virt-about-vms-and-boot-sources.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc
+// * virt/vm_templates/virt-creating-vm-template.adoc
+
+[id="virt-about-vms-and-boot-sources_{context}"]
+
+= About virtual machines and boot sources
+
+Virtual machines consist of a virtual machine definition and one or more disks, backed by data volumes.
+
+Virtual machine templates enable you to create virtual machines using preset virtual machine definitions.
+
+To use the boot sources feature, you must install the latest release of {VirtProductName}. The namespace `openshift-virtualization-os-images` enables the feature and is installed with the {VirtProductName} Operator.
+
+Every virtual machine template requires a boot source, which is a fully configured virtual machine disk image including configured drivers. Each virtual machine template contains a virtual machine definition with a pointer to the boot source. Each boot source has a predefined name and namespace. For many operating systems, a boot source is automatically provided. If it is not provided, then a custom boot source must be prepared by the administrator.
+
+After preparing and uploading boot source image files, you create configured virtual machines by attaching the boot source to a virtual machine template.
+
+You attach Boot sources to templates using the Web Console. You can define a Boot sources using a persistent volume claim (PVC) populated by uploading a local file, cloning an existing PVC, importing by registry, or from a local file. Once the boot source is attached to a virtual machine template, you create any number of full-configured ready-to-use virtual machines from the template.
```

Comment:
"Boot" to "boot"? in the sections before, "Boot sources" got spelled as "boot sources" (lower case b)

Reply:
Thanks, fixed.